package.json

```diff
@@ -76,7 +76,8 @@
   "dependencies": {
     "@babel/plugin-transform-object-assign": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
-    "convert-source-map": "^1.7.0",
+    "@types/convert-source-map": "^2.0.0",
```

Comment:
I think we can move this to `devDependencies`.

Reply:
You are right, I will move it.